jdbcSource/src/test/java/io/vantiq/extsrc/jdbcSource/TestJDBC.java

```diff
@@ -1180,6 +1245,36 @@ public static void setupSource(Map<String, Object> sourceDef) {
             core = new JDBCCore(testSourceName, testAuthToken, testVantiqServer);
             core.start(CORE_START_TIMEOUT);
         }
+        if (insertResponse.hasErrors()) {
+            log.debug("Errors creating source: {}", insertResponse.getErrors());
+        }
+        assert insertResponse.isSuccess();
+    
```

Comment:
When we start the connector, wait for it to be fully up.  There are still some issues where the connector is repeatedly started (I think the server is processing the _now it's there, now it's not_ source.